cmd/observer/observer/handshake.go

```
@@ -198,20 +207,32 @@ func Handshake(
 	}
 
 	var statusMessage StatusMessage
-	if err := readMessage(conn, 16+eth.StatusMsg, HandshakeErrorIDStatusDecode, &statusMessage); err != nil {
-		return &helloMessage, nil, err
+	if err := readMessage(conn, 16+eth.StatusMsg, HandshakeErrorIDStatusDecode, &statusMessage); err == nil {
```

the err here gets swallowed which is not good - can you please log it and say it falls back to trying to read eth69 msg